httpcore5/src/main/java/org/apache/hc/core5/http/HttpHeaders.java

```diff
@@ -130,6 +188,17 @@ private HttpHeaders() {
 
     public static final String STATUS_URI = "Status-URI";
 
+    /**
+     * The HTTP {@code Set-Cookie} header field name.
+     * @see <a href="https://tools.ietf.org/html/rfc2109#section-4.2.2">Section 4.2.2 of RFC 2109</a>
+     */
+    public static final String SET_COOKIE = "Set-Cookie";
+    /**
+     * The HTTP {@code Set-Cookie2} header field name.
+     * @see <a href="https://tools.ietf.org/html/rfc2965">RFC 2965</a>
+     */
+    public static final String SET_COOKIE2 = "Set-Cookie2";
```

Comment:
@arturobernalg `Set-Cookie2` is considered obsolete and should not be used. I would not include it,

Reply:
Agree, changed